app/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
     }
     defaultConfig {
         applicationId "de.tum.in.tca.ticketcheck"
-        minSdkVersion 24
+        minSdkVersion 21
```

Comment:
We don't need 21 here. Android 6/7 should be just fine as most devices use that.

Reply:
Then let us do 23. I could not run the app on my phone with Android 6.0.